### DISCUSSION TITLE
[ BUG ] Version history records not showing more than 10 on data objects

### DISCUSSION THREAD

**Discussion by Karthi96** [pimcore/pimcore#9552] [Q&A / Support] [answered]
Hi Team, 
How can we increase the version history count of data objects? Though I have enabled 50 steps to be store on version but in the version tab, I can see only 10 histories, why is that so?

![Screenshot from 2021-06-24 08-27-58](https://user-images.githubusercontent.com/17947667/123195795-69e28c80-d4c6-11eb-80f5-7cea9aca9639.png)
 

Thanks, 
Karthikeyan B

**Comment by dvesh3** [comment, marked as answer]
sorry, it works for me:
![image](https://user-images.githubusercontent.com/5137917/123228366-ac5f9580-d4d5-11eb-8c85-1a2c4950336f.png)
after running maintenance job "versioncleanup"
![image](https://user-images.githubusercontent.com/5137917/123228390-b1bce000-d4d5-11eb-80ac-32baf1417cf1.png)
